mod/gcforums/lib/functions.php

```diff
@@ -0,0 +1 @@
+<?php
```

Comment:
I was in the middle of cleaning up (file structure and code) the plugin, but had to switch over to a different issue with a higher priority to fix.